orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -113,8 +137,12 @@ def get_buffer(self, target: str) -> torch.Tensor:
         return self._original_module.get_buffer(target)
 
     def parameters(self, recurse: bool = True) -> Iterator[torch.nn.Parameter]:
-        """Override original method to delegate execution to the base module"""
-        yield from self._original_module.parameters(recurse=recurse)
+        if self._onnx_model:
+            for _, parameter in self._onnx_model_parameters:
+                yield parameter
+        else:
+            """Override original method to delegate execution to the base module"""
+            yield from self._original_module.parameters(recurse=recurse)
```

Comment:
you would have to override all `torch.nn.Module` APIs to prevent runtime errors

Reply:
corrected, please check